tools/emcoverage.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
Does it need python3?    Currently all our tools are running as "/usr/bin/env python" and support both

Reply:
The way this tool is written, only one python version can possibly be supported, since we execute the coverage magic inside the same process. I would rather support python 3 over python 2 in this situation, especially since it would be nice to run the CI tests under coverage, and we use `python3` on CI.

An alternative is to create two distinct entry points, `emcoverage2` and `emcoverage3`, that does `import os, emcoverage; sys.executable = os.path.abspath(__file__); emcoverage.main()`.